docs/config.md

```diff
@@ -0,0 +1,43 @@
+# Configuration
```

Comment:
This is mainly just moved content from the old [setup.md file](https://github.com/mozilla-services/syncstorage-rs/blob/master/docs/setup.md)